golang/go-cloud-run-hello-world/cloudbuild.yaml

```diff
@@ -0,0 +1,67 @@
+steps:
+
+- id: 'Build Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['build', '-t', 'gcr.io/${PROJECT_ID}/${_SERVICE_TAG}:${COMMIT_SHA}', '.']
+
+- id: 'Push Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['push', 'gcr.io/${PROJECT_ID}/${_SERVICE_TAG}:${COMMIT_SHA}']
+
+- id: 'Deploy to Cloud Run'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  args:
+  - 'run'
+  - 'deploy'
+  - '${_SERVICE}'
+  - '--image'
+  - 'gcr.io/${PROJECT_ID}/${_SERVICE_TAG}:${COMMIT_SHA}'
+  - '--allow-unauthenticated'
+  - '--region'
+  - '${_CLOUDSDK_REGION}'
+  - '--platform'
+  - 'managed'
+
+- id: 'Get Cloud Run URL'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    get_url() {
+        gcloud run services describe ${_SERVICE} --format 'value(status.url)' \
+          --platform managed --region ${_CLOUDSDK_REGION}
+    }
+    echo $(get_url) > _service_url
+    echo "Cloud Run URL for ${_SERVICE} is $(cat _service_url)"
```

Comment:
This was adapted from the `go-hello-world` sample. Since the `test_content.sh` is no longer trying to be general-purpose this could be moved into it to simplify the `cloudbuild.yaml`.